frontend/src/modules/Vfp/loadModule.tsx

```diff
@@ -0,0 +1,14 @@
+import { ModuleRegistry } from "@framework/ModuleRegistry";
+
+
+import { MODULE_NAME } from "./registerModule";
+import { Interface, State, interfaceInitialization } from "./state";
+import { View } from "./view";
+import { Settings } from "./settings/settings";
+
+const defaultState: State = {};
+
+const module = ModuleRegistry.initModule<State, Interface>(MODULE_NAME, defaultState, {}, interfaceInitialization);
```

Comment:
There are some small changes when initializing a module, which causes CI to fail.
See examples in this pr:
https://github.com/equinor/webviz/pull/679/files#diff-fe37b3b10b1a6a2646514d0468abd0e6883f49dc29990f4a2b28d42fb60fee1f